.github/workflows/new-release.yml

```diff
@@ -0,0 +1,66 @@
+name: New Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        type: string
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        cache: 'yarn'
+    - name: Get version # TODO(eeeqeee) add option to increment from current version
+      id: get-version
+      run: |
+        echo "version=${{ inputs.version }}" >> $GITHUB_OUTPUT
+    - name: Set git config
+      run: |
+        git config user.name "github-actions[bot]"
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where does this come from? is it just standard? You seem pretty safe with exposing info, but just wanna make sure it's not some repo-specific thing that leaks info somehow to have this visible publicly

Reply:
https://github.com/orgs/community/discussions/26560
Its the name+email for the Github Action bot